lib/process-services/src/lib/form/widgets/content-widget/attach-file-widget.component.ts

```diff
@@ -217,7 +217,7 @@ export class AttachFileWidgetComponent extends UploadWidgetComponent implements
     private isExternalHost(repository: AlfrescoEndpointRepresentation): boolean {
         const currentECMHost = this.getDomainHost(this.appConfigService.get(AppConfigValues.ECMHOST));
         const chosenRepositoryHost = this.getDomainHost(repository.repositoryUrl);
-        return chosenRepositoryHost !== currentECMHost;
+        return !this.appConfigService.get(AppConfigValues.DEVMODE) && chosenRepositoryHost !== currentECMHost;
```

Comment:
you should be able already to detect the dev/production mode via native angular support
https://www.digitalocean.com/community/tutorials/angular-environment-variables#detecting-the-environment